ktlint-ruleset-experimental/src/main/kotlin/com/pinterest/ktlint/ruleset/experimental/IndentationRule.kt

```diff
@@ -208,11 +208,11 @@ class IndentationRule : Rule("indent"), Rule.Modifier.RestrictToRootLast {
             return
         }
         if (!node.nextCodeLeaf()?.prevLeaf {
-                // Skip comments, whitespace, and empty nodes
-                !it.isPartOfComment() &&
+            // Skip comments, whitespace, and empty nodes
+            !it.isPartOfComment() &&
                 !it.isWhiteSpaceWithoutNewline() &&
```

Comment:
This formatting looks weird to me, but IntelliJ does the same thing.

Reply:
looks right; it's a continuation indent.